GVFS/GVFS.FunctionalTests/Tools/GVFSFunctionalTestEnlistment.cs

```diff
@@ -233,9 +234,30 @@ public void UnmountAndDeleteAll()
             this.DeleteEnlistment();
         }
 
-        public string GetVirtualPathTo(params string[] pathInRepo)
+        public string GetVirtualPathTo(string path)
         {
-            return Path.Combine(this.RepoRoot, Path.Combine(pathInRepo));
+            string cleanedPath;
+            switch (Path.DirectorySeparatorChar)
+            {
+                case '/':
+                    cleanedPath = path.Replace('\\', Path.DirectorySeparatorChar);

```

Comment:
This line is a bit worrisome - it seems like it could hide tests that are still assuming Windows slashes, rather than force us to clean up the tests.

I would expect this method to convert from Git paths to file system paths, but not to cover over platform-related differences. So to me, that means that it should do a conversion on Windows, and noop on Mac. Thoughts?

Reply:
@sanoursa I went back and forth on this as well.  I agree with your assessment that we might miss cleaning up tests with this change.  I'll switch this to just convert Git path to system path.